source/tomopy/util/extern.py

```diff
@@ -126,6 +125,12 @@ def c_shared_lib(lib_name, do_warn=True):
 LIB_TOMOPY_GRIDREC = c_shared_lib("libtomopy-gridrec")
 
 
+def MissingLibrary():
+    raise ModuleNotFoundError(
+        '''Function does not exist due to missing library.
+        Check CMake log for more details.''')
```

Comment:
This error message is too generic, and triple quotations enforces the line breaks and whitespace.

Instead something like f"The {function}" algorithm is not available because its compiled library is missing. Check whether TomoPy was compiled with the dependencies necessary for this algorithm." not only tells us what the problem is, but it suggests a solution.